<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page1</class>
 <widget class="QWidget" name="page1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDR</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="page1_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1181</width>
     <height>881</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 58, 89);
border-radius: 100px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>501</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);
border-radius: 0x;</string>
    </property>
    <property name="text">
     <string>이미지를 넣어주세요.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="img_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>730</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 고딕 Std B</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 28, 43);
border: rgb(134, 140, 214);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;
border-top-left-radius: 8px;
color: rgb(203, 203, 203);

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(194, 194, 194);
}
</string>
    </property>
    <property name="text">
     <string>select</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="time">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 고딕 Std B</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 200, 200);</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1753</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd  hh:mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="description_img">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>780</y>
      <width>361</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(200, 200, 200);
</string>
    </property>
    <property name="text">
     <string>upload original x-ray image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:rgb(255, 216, 247);
border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>720</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 고딕 Std B</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(50, 50, 50);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;
border-top-left-radius: 8px;

color: rgb(56, 58, 89);
background-color:rgb(255, 216, 247);

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background: rgb(255, 85, 255);

}</string>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="the_result">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:None;
color: rgb(56, 58, 89);
color: rgb(255, 216, 247);
</string>
    </property>
    <property name="text">
     <string> Test  Result </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pred">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>400</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>23</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:none;
border-radius: 0x;
color: rgb(108, 113, 182);

padding-bottom: 7px;
background-color: rgb(255, 255, 255);

border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;

border: 2px solid rgb(134, 140, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Pred</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_name">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:none;
border-radius: 0x;
color: rgb(108, 113, 182);
padding-bottom: 7px;
background-color: rgb(255, 255, 255);

border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>720</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 고딕 Std B</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(50, 50, 50);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;
border-top-left-radius: 8px;

color: rgb(56, 58, 89);
background-color:rgb(255, 216, 247);

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
    background: rgb(255, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:rgb(255, 216, 247);
color: rgb(56, 58, 89);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#757575;&quot;&gt;* 환자번호를 입력해 주세요.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>310</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:None;
border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;

border: 5px solid rgb(255, 216, 247);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="normal_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:none;
border-radius: 0x;
color: rgb(108, 113, 182);

padding-bottom: 7px;
background-color: rgb(255, 255, 255);

border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;

border: 2px solid rgb(134, 140, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>normal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nn_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>560</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:none;
border-radius: 0x;
color: rgb(108, 113, 182);

padding-bottom: 7px;
background-color: rgb(255, 255, 255);

border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;

border: 2px solid rgb(134, 140, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>not normal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pneu_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>620</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:none;
border-radius: 0x;
color: rgb(108, 113, 182);

padding-bottom: 7px;
background-color: rgb(255, 255, 255);

border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;

border: 2px solid rgb(134, 140, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Pneumonia</string>
    </property>
   </widget>
   <zorder>label1_5</zorder>
   <zorder>label</zorder>
   <zorder>img_button</zorder>
   <zorder>time</zorder>
   <zorder>description_img</zorder>
   <zorder>label1</zorder>
   <zorder>pushButton</zorder>
   <zorder>the_result</zorder>
   <zorder>pred</zorder>
   <zorder>input_name</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_2</zorder>
   <zorder>normal_label</zorder>
   <zorder>nn_label</zorder>
   <zorder>pneu_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>page1</receiver>
   <slot>show_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1052</x>
     <y>730</y>
    </hint>
    <hint type="destinationlabel">
     <x>1106</x>
     <y>932</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>img_button</sender>
   <signal>clicked()</signal>
   <receiver>page1</receiver>
   <slot>filedialog_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>777</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>929</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>page1</receiver>
   <slot>save_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>917</x>
     <y>730</y>
    </hint>
    <hint type="destinationlabel">
     <x>922</x>
     <y>933</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>page1</receiver>
   <slot>info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>1000</x>
     <y>-25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_exit()</slot>
  <slot>filedialog_open()</slot>
  <slot>save_data()</slot>
  <slot>info()</slot>
 </slots>
</ui>
